cocos/2d/framework/ui-renderer.ts

```diff
@@ -498,7 +529,6 @@ export class UIRenderer extends Renderer {
         this.node._uiProps.colorDirty = true;
         this.setEntityColorDirty(true);
         this.setEntityColor(this._color);
-        this.setEntityOpacity(this.node._uiProps.localOpacity);
```

Comment:
`this.node._uiProps.localOpacity` has already been synchronized to native, doesn't need to invoke `setEntityOpacity` here.